client/client.js

```diff
@@ -3,23 +3,53 @@ var assert = require('assert');
 var boot = require('loopback-boot');
 var debug = require('debug')('strong-mesh-models:client');
 var loopback = require('loopback');
-var request = require('request');
+var serverService = require('./models/server-service');
+var serviceInstance = require('./models/service-instance');
 var url = require('url');
+var urlDefaults = require('strong-url-defaults');
 var util = require('util');
+var path = require('path');
+
+/**
+ * Mesh API Client
+ *
+ * @param {string} apiUrl URL for Mesh server. Formats supported include:
+ *   * http+unix:///full/path/to/socket
```

Comment:
I'd like this to support partial-socket paths, with no protocol scheme, as well. Right now some of that normalization is in callers of the client.